packages/node_modules/@webex/internal-plugin-devices/test/unit/spec/features/feature-model.js

```diff
@@ -56,6 +56,113 @@ describe('plugin-devices', () => {
       });
     });
 
+    describe('#parse()', () => {
+      let fixture;
+      let model;
+
+      beforeEach('initialize the feature model', () => {
+        featureModel = new FeatureModel();
+      });
+
+      it('should return an empty object when the model is not defined', () => {
+        model = featureModel.parse(fixture);
+        assert.deepEqual(model, {});
+      });
+
+      describe('when the model is defined', () => {
+        beforeEach('define the fixture', () => {
+          fixture = {};
+        });
+
+        describe('when the value is a number', () => {
+          let num;
+
+          beforeEach('set the value to a number', () => {
+            fixture.val = '1234';
+            num = Number(fixture.val);
```

Comment:
Since this value is only used in the `it('should set the value to match the pto', () =>`, let's just define it in that test block.

Reply:
Moved the `num = Number(fixture.val);` to it's appropriate `it` clause.